src/pages/Goods/Goods.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from "react";
+import "./Goods.scss";
+
+const Goods = () => {
+  const [quantity, setQuantity] = useState(1);
+  const [isActive, setisActive] = useState(true);
+
+  const quantityHandler = ({ target }) => {
+    target.className === "quantityUp"
+      ? setQuantity(quantity + 1)
+      : quantity > 0 && setQuantity(quantity - 1);
+  };
```

Comment:
```suggestion
  const increaseQuantity = ( ) => { }
  const decreaseQuantity = ( ) => { }
```

* key className이 변경되면 함수 조건도 변경되야 해서 유지보수에 좋지않습니다.
* 한가지 함수는 한가지 기능만 하도록 분리하는것이 유지보수와 가독성에 좋습니다.
* 이 함수에는 **수량을 늘리는 기능**과, **수량을 줄이는 기능** 모두 있는데, 둘을 나누는게 좋겠습니다.